.github/workflows/ci-test-xtgeo-cibuildwheel.yml

```diff
@@ -0,0 +1,147 @@
+# Test building some cibuildwheel on pull requests (simplified testing)
```

Comment:
This comment.. This file isn't just for testing the setup of CI pipeline though? If that's the case this must be removed

Reply:
Removed :+1: